plugin/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -147,9 +148,24 @@ const fetchNodeHtmlImageMediaItemNodes = async ({
 
         sourceUrl = ensureSrcHasHostname({ wpUrl, src: sourceUrl })
 
+        const existingNode = helpers.getNode(id)
+
+        if (!existingNode) {
+          if (!displayedFailedToRestoreMessage) {
+            helpers.reporter.warn(
+              formatLogMessage(
+                `File node failed to restore from cache. This is a bug in gatsby-source-wordpress. Please open an issue so we can help you out :)`
```

Comment:
totally random, but how does Gatsby handle translations for strings like this? Is there something similar to WordPress `__()` functions? Or has Gatsby not gotten that far?

Reply:
Yeah, Gatsby hasn't gotten that far yet